lib/ansible/modules/crypto/luks_device.py

```diff
@@ -197,6 +234,30 @@ def __init__(self, module):
     def _run_command(self, command):
         return self._module.run_command(command)
 
+    def get_device_by_uuid(self, uuid):
+        ''' Returns the device that holds UUID passed by user
+        '''
+        self._blkid_bin = self._module.get_bin_path('blkid', True)
+        uuid = self._module.params.get('uuid')
```

Comment:
The
```suggestion
        uuid = self._module.params['uuid']
```
you had here before is also fine. `AnsibleModule` makes sure that all options will be there; if the user didn't specify them, you'll get `None`.

Reply:
I got some errors without the .get method on `test_luks_device.py`, not on this place exactly. dont know why but they are there, well, I change this line and the another on label.